docs/document/website/versioned_docs/version-1.10.0/components/desktop/dialog.md

```diff
@@ -0,0 +1,116 @@
+---
+id: version-1.10.0-dialog
+title: Dialog
+sidebar_label: Dialog
+original_id: dialog
+---
+
+## Overview
+
+The Dialog component displays a dialog box.
+
+<div class="sample-container" id="dialog">
+  <div id="sample-container__components">
+    <iframe id="iframe" title="dialog image" width="700px" height="300px"></iframe>
+  </div>
+</div>
+<script src="/js/samples/desktop/dialog.js"></script>
+
+---
+
+## Specification
+
+### Property
+
+Here is a list of properties that can be used for modifying the component:
+
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| title | string | ""  | Header Title | |
+| content *1 | string/HTMLElement | ""  | DOM inside content | If a string with HTML is set, it will be automatically converted to HTML and displayed as it is |
+| footer *1 | string/HTMLElement | ""  | DOM inside footer | If a string with HTML is set, it will be automatically converted to HTML and displayed as it is |
+
+> *1: [Security] Kintone UI Component does NOT sanitize this property value. It is the developer's responsibility to escape any user input when using this option so that XSS attacks would be prevented.
+
+### Event
+
+Here is a list of events that can be specified:
+
+| Name | Type | Description | Remark |
+| :--- | :--- | :--- | :--- |
+| close | function | Event handler when the component has been closed | It will pass the event object as the argument |
+
+### Constructor
+
+Dialog(options)<br>
+Here is a list of available constructors:
+
+#### Parameter
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| options | object | {} | Object that includes component properties | |
+
+---
+### Method
+
+Here is a list of available methods:
+
+#### open()
+Show the Dialog
+
+##### Parameter
+none
+
+##### Return
+none
+
+#### close()
+Hide the Dialog
+
+##### Parameter
+none
+
+##### Return
+none
+
+---
+## Sample Code
+
+Here is a sample code when all parameters are specified:
+
+```javascript
+// Create Ok and Cancel buttons
+const dialogOKButton = new Kuc.Button({
+  text: 'OK',
+  type: 'submit'
+});
+const dialogCancelButton = new Kuc.Button({
+  text: 'Cancel',
+  type: 'cancel',
+  id: 'kuc_dialog_cancel_button'
+});
+dialogOKButton.addEventListener('click', () => {
+    // handle click Ok button
+})
+dialogCancelButton.addEventListener('click', () => {
+    // handle click Cancel button
+})
+
+// Wrap Ok and Cancel buttons into a div
+const divEl = document.createElement('div');
+divEl.appendChild(dialogOKButton)
+divEl.appendChild(dialogCancelButton)
+
+const dialog = new Kuc.Dialog({
+  title:  'Title',
+  content: '<div>This is Content</div>',
+  footer: divEl
+});
+
+dialog.addEventListener('close', event => {
+  console.log(event);
+});
+
+dialog.open();
+dialog.close();
```

Comment:
- Please update the doc directory's md file, too.
- Here is my suggestion. Please check it.
  - I added `const Kuc = Kucs['1.x.x'];` line.
  - I found a mistake on type prop, so I modified.
  - I found icon prop is missing, so I added.

```suggestion
const Kuc = Kucs['1.x.x'];

// Create OK and Cancel buttons
const okButton = new Kuc.Button({
  text: 'OK',
  type: 'submit'
});
const cancelButton = new Kuc.Button({
  text: 'Cancel',
  type: 'normal',
});

okButton.addEventListener('click', () => {
    // handle click OK button
})
cancelButton.addEventListener('click', () => {
    // handle click Cancel button
})

// Wrap OK and Cancel buttons with a div
const divEl = document.createElement('div');
divEl.appendChild(okButton)
divEl.appendChild(cancelButton)

const dialog = new Kuc.Dialog({
  title:  'Title',
  content: '<div>This is Content</div>',
  footer: divEl,
  icon: 'info'
});

dialog.addEventListener('close', event => {
  console.log(event);
});

dialog.open();
dialog.close();
```